arm:
  arm_joint_controller:
    type: position_controllers/JointTrajectoryController
    joints:
      - zhixianmove
      - zhuan1
      - zhuan2
      - zhuan3
      - zhuan4
      - zhuan5
    gains:
      zhixianmove:
        p: 1.0
        d: 0.0
        i: 0.0
        i_clamp: 1
      zhuan1:
        p: 1.0
        d: 0.0
        i: 0.0
        i_clamp: 1
      zhuan2:
        p: 1.0
        d: 0.0
        i: 0.0
        i_clamp: 1
      zhuan3:
        p: 1.0
        d: 0.0
        i: 0.0
        i_clamp: 1
      zhuan4:
        p: 1.0
        d: 0.0
        i: 0.0
        i_clamp: 1
      zhuan5:
        p: 1.0
        d: 0.0
        i: 0.0
        i_clamp: 1

  gazebo_ros_control:
    pid_gains:
      zhixianmove:
        p: 800.0
        d: 180.0
        i: 40.0
        i_clamp: 100
      zhuan1:
        p: 200
        d: 40
        i: 20
        i_clamp: 100
      zhuan2:
        p: 50.0
        d: 10.0
        i: 3
        i_clamp: 100
      zhuan3:
        p: 30
        d: 5
        i: 0.5
        i_clamp: 100
      zhuan4:
        p: 20
        d: 2
        i: 0.2
        i_clamp: 100
      zhuan5:
        p: 1
        d: 0.0
        i: 1
        i_clamp: 20